docs/guides/offline-cli.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+---
+
+In this guide, we'll learn how to use Meyda's command line interface to perform
+offline audio feature extraction on a file on disk.
+
+The difference between online and offline audio feature extraction is that while
+online audio feature extraction happens in "real time", while audio is playing
+to a listener, offline audio feature extraction doesn't output the audio. As a
+result, it can run much faster than real time, because it doesn't need to wait
+for the audio to be played out of the speakers, and can run through an audio
```

Comment:
Actually offline analysis mainly doesn't need to wait for new input buffers from the ADC. Don't forget we can perform real-time feature extraction without playing the audio back through the speakers at all. I guess the main difference between online/offline is that:
* online: performing feature extraction on audio that is being generated in real-time. For example, a user speaking to a microphone connected to the ADC.
* offline: performing feature extraction on audio that was generated before, and is saved to disk. For example, a recording of a user speaking.

Reply:
Updated 😄